src/Framework/Microsoft.AspNetCore.App/Microsoft.AspNetCore.App.props

```diff
@@ -45,13 +39,9 @@
     <Dependency Include="Microsoft.AspNetCore.Http"                                    Version="$(MicrosoftAspNetCoreHttpPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.HttpOverrides"                           Version="$(MicrosoftAspNetCoreHttpOverridesPackageVersion)" />
     <Dependency Include="Microsoft.AspNetCore.HttpsPolicy"                             Version="$(MicrosoftAspNetCoreHttpsPolicyPackageVersion)" />
-    <Dependency Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore"            Version="$(MicrosoftAspNetCoreIdentityEntityFrameworkCorePackageVersion)" />
-    <Dependency Include="Microsoft.AspNetCore.Identity.UI"                             Version="$(MicrosoftAspNetCoreIdentityUIPackageVersion)" />
```

Comment:
cc @javiercn

Reply:
@natemcmaster What's the reasoning for removing the identity UI package from the shared framework?

Reply:
When we reviewed all assemblies for 3.0 against [these guidelines](https://github.com/aspnet/AspNetCore/blob/master/docs/SharedFramework.md) with @DamianEdwards @davidfowl @Eilon  and others, Identity.UI did not meet the criteria to be in the shared framework. 

See also design meeting notes: https://github.com/aspnet/specs/blob/master/runtime/design-notes/2018-07-13-aspnetcore-3.0-shared-fx.md